### PR TITLE
Notes for EP28

### PR DESCRIPTION
**Pull request by Temo27anas**

### CODE REVIEW

File: blablas/ep28/index.md [hunk 1]
```diff
@@ -8,21 +8,56 @@ isNext: false
...
+
+00:59 - connected devices (Q:Ayman A:Achraf)
+
+01:02 - Do you have plans for your studies (the program) ?
+
+01:07 - Facebook Questions
+
+01:17 - what do you expect your parents to improve your technical skills?
+
+01:27 - what do you advise kids in your age to start programming?
```
Comment (yjose) [inline blablas/ep28/index.md:53]: What your advice for kids in your age to start programming?

File: blablas/ep28/index.md [hunk 2]
```diff
@@ -8,21 +8,56 @@ isNext: false
...
+
+00:46 - What are the steps to take when you want to create a new project?
+
+00:56 - Are you thinking about new projects in the future?
+
+00:59 - connected devices (Q:Ayman A:Achraf)
+
+01:02 - Do you have plans for your studies (the program) ?
+
+01:07 - Facebook Questions
```
Comment (yjose) [inline blablas/ep28/index.md:49]: QA instead of facebook questions

File: blablas/ep28/index.md [hunk 3]
```diff
@@ -8,21 +8,56 @@ isNext: false
...
+
+00:25 - How do you like the field of information technology?
+
+00:29 - How did your family and friends react to you when they know you love programming?
+
+00:34 - have you an idea to explain your knowledge of IT in Darija to your friends , your family , and others  kids ?
+
+00:41 - How did you get the idea for your projects?
+
+00:46 - What are the steps to take when you want to create a new project?
```
Comment (yjose) [inline blablas/ep28/index.md:41]: Your workflow creating new projects?

File: blablas/ep28/index.md [hunk 4]
```diff
@@ -8,21 +8,56 @@ isNext: false
...
+
+00:16 - how they improved their skills?
+
+00:20 - Have you ever practiced these skills in competition?
+
+00:25 - How do you like the field of information technology?
+
+00:29 - How did your family and friends react to you when they know you love programming?
+
+00:34 - have you an idea to explain your knowledge of IT in Darija to your friends , your family , and others  kids ?
```
Comment (yjose) [inline blablas/ep28/index.md:37]: Do you have some plans to explain your IT knowledge in Darija ....

File: blablas/ep28/index.md [hunk 5]
```diff
@@ -8,21 +8,56 @@ isNext: false
...
+
+01:17 - What do you expect your parents to improve your technical skills?
+
+01:27 - What your advice for kids in your age to start programming?
+
+01:31 - Final words and goodbye.
+
 
 ## Links
+[Scratch](http://scratch.mit.edu/)
```
Comment (yjose) [inline blablas/ep28/index.md:59]: Can you add a line break between links 🙏

Comment (Temo27anas) [reply blablas/ep28/index.md]: DONE

**Pull request merged**